dashboard/app/assets/javascripts/header.js.erb

```diff
@@ -251,7 +251,7 @@ function shareProject() {
   });
 }
 
-function remixProject() {
+function remixLegacyShare() {
```

Comment:
I'm not sure I know what legacy /c/ share pages are, but I wonder if we're overloading the term legacy (referring to both these /c/ shares and and the /p/applab#channel_id urls). Not sure what a good differentiator is without better understanding /c/.